### DISCUSSION TITLE
Could not transfer artifact org.apache.camel:camel-bom:pom:4.1.0 from/to central (https://repo1.maven.org/maven2): PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested

### DISCUSSION THREAD

**Discussion by nmvprasad** [apache/camel-karavan#965] [General]
I try to follow these steps to install and run Karavan on my local using docker
https://github.com/apache/camel-karavan/blob/main/docs/WEB_DOCKER.md

looks like karavan is build with OpenJDK java image and it is not able to download due certificate from this mavan repository.
So not sure how to add this certificate or force to sue different jdk which is there on my local for karavan

https://repo1.maven.org/maven2): PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target

Docker CL:

java -version
openjdk version "17.0.8.1" 2023-08-24
OpenJDK Runtime Environment Temurin-17.0.8.1+1 (build 17.0.8.1+1)
OpenJDK 64-Bit Server VM Temurin-17.0.8.1+1 (build 17.0.8.1+1, mixed mode, sharing)

echo $JAVA_HOME
/opt/java/openjdk

**Comment by mgubaidullin** [comment]
On what step from https://github.com/apache/camel-karavan/blob/main/docs/WEB_DOCKER.md
you got the error?

**Comment by davsclaus** [comment]
closing as user did not respond
